queries/python.scm

```diff
@@ -626,9 +626,9 @@
 ) @argumentOrParameter.iteration.domain
 
 (argument_list
-  "(" @argumentOrParameter.iteration.start.endOf
-  ")" @argumentOrParameter.iteration.end.startOf
-) @argumentOrParameter.iteration.domain
+  "(" @argumentOrParameter.iteration.start.endOf @name.iteration.start.endOf @value.iteration.start.endOf
```

Comment:
NB: Andreas says there's a limit of three captures on each node and more are silently discarded by tree-sitter, so beware.

Reply:
Ideally we should lint this - or adding to SCM tests